integration_tests/dbt_project.yml

```diff
@@ -9,64 +9,16 @@ dispatch:
 
 vars:
   klaviyo_schema: klaviyo_integration_tests
-  klaviyo_source:
+  klaviyo_source: 
     klaviyo_campaign_identifier: "campaign"
     klaviyo_event_identifier: "event"
     klaviyo_flow_identifier: "flow"
     klaviyo_integration_identifier: "integration"
     klaviyo_person_identifier: "person"
     klaviyo_metric_identifier: "metric"
-    klaviyo__event_pass_through_columns: []
-    klaviyo__person_pass_through_columns: []
 
-  klaviyo:
-    event_table: "{{ ref( 'stg_klaviyo__event') }}" 
-    campaign: "{{ ref( 'stg_klaviyo__campaign') }}" 
-    flow: "{{ ref( 'stg_klaviyo__flow') }}" 
-    integration: "{{ ref( 'stg_klaviyo__integration') }}" 
-    person: "{{ ref( 'stg_klaviyo__person') }}" 
-    metric: "{{ ref( 'stg_klaviyo__metric') }}" 
-    klaviyo__email_attribution_lookback: 120 # in hours - 5 days -> https://help.klaviyo.com/hc/en-us/articles/115005248128#conversion-tracking1
-    klaviyo__sms_attribution_lookback: 24 # in hours -> https://help.klaviyo.com/hc/en-us/articles/115005248128#sms-conversion-tracking7
-
-    klaviyo__eligible_attribution_events: # default in klaviyo -> https://help.klaviyo.com/hc/en-us/articles/115005248128 
-      - 'opened email'
-      - 'clicked email'
-      - 'clicked sms'
-
-    klaviyo__count_metrics: 
-      - 'Active on Site' # default API metrics
-      - 'Viewed Product'
-      - 'Ordered Product' # default Shopify metrics
-      - 'Placed Order'
-      - 'Refunded Order' # note: klaviyo only tracks FULL refunds (not partial)
-      - 'Cancelled Order'
-      - 'Fulfilled Order'
-      - 'Received Email' # default email and sms conversion metrics in Klaviyo
-      - 'Clicked Email'
-      - 'Opened Email'
-      - 'Bounced Email'
-      - 'Marked Email as Spam'
-      - 'Dropped Email'
-      - 'Subscribed to List'
-      - 'Unsubscribed to List'
-      - 'Unsubscribed'
-      - 'Updated Email Preferences'
-      - 'Subscribed to Back in Stock'
-      - 'Merged Profile'
-      - 'Received SMS'
-      - 'Clicked SMS'
-      - 'Consented to Receive SMS'
-      - 'Sent SMS'
-      - 'Unsubscribed from SMS'
-      - 'Failed to Deliver SMS'
-
-    klaviyo__sum_revenue_metrics:
-      - 'Refunded Order'
-      - 'Placed Order'
-      - 'Ordered Product'
-      - 'checkout started'
-      - 'cancelled order'
```

Comment:
All of this is already defined in the `dbt_project.yml` file of the package itself. Restating it here is duplicative and is not necessary.